src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -793,7 +793,7 @@ public function dropForeignKey($tableName, $columns, $constraint = null)
     /**
      * {@inheritdoc}
      */
-    public function getSqlType($type, $limit = null)
+    public function getSqlType($type, $limit = null, $identity = false)
```

Comment:
I don't recommend modifying this signature, as it is defined by the AdapterInterface.